Library/Homebrew/cmd/update.sh

```diff
@@ -309,6 +314,11 @@ EOS
     set -x
   fi
 
+  if [[ -z "$HOMEBREW_AUTO_UPDATE_SECS" ]]
+  then
+    HOMEBREW_AUTO_UPDATE_SECS="60"
+  fi
```

Comment:
Will leave it as-is for now just because I think it's a bit more explicit. 👍 to maybe fixing such things in bulk in future though.